services/121-service/src/registration/dto/create-registration-distinctness.dto.ts

```diff
@@ -0,0 +1,20 @@
+import {
+  ArrayMaxSize,
+  ArrayMinSize,
+  IsArray,
+  IsNotEmpty,
+  IsNumber,
+  IsString,
+} from 'class-validator';
+
+export class CreateRegistrationDistinctnessDto {
+  @IsArray()
+  @ArrayMinSize(2)
+  @ArrayMaxSize(10)
```

Comment:
Would this number need to be 'connected' to the value here:
https://github.com/global-121/121-platform/pull/6636/files#diff-bc34a1b089a694f457575110ff9fab59b83e69b7822bb7af50d898048d5a186dR180

Also, why is one of those `10` and the other `9`? Is it because the `9` refers to the "others", when comparing a specific Registration?

We could have a 'central' constant somewhere to set the "maximum number of duplicates"; But I don't have a clear suggestion to "where to store such a number"... :)

Reply:
resolved with rebase